features/config/upgrade/src/main/resources/changelog-cm/30.0.0/changelog-cm.xml

```diff
@@ -76,4 +76,11 @@
         </cm:changeSchema>
         <cm:importConfig schemaId="org.opennms.features.topology.app.icons.application" configId="default" filePath="org.opennms.features.topology.app.icons.application.cfg"/>
     </changeSet>
+
+    <!--TODO check which schemas can have multiple configurations-->
```

Comment:
Open TODO?

Reply:
Ok. As as discussed I will consider all existing at this time configurations as non-multiple.